tests.html

```diff
@@ -43,6 +43,12 @@ <h2 id="g1-title"></h2>
                     <canvas id="g1" style="width: 100%; height: 256px"></canvas>
                 </div>
             </div>
+            <div class="row">
```

Comment:
Commit "Show averge time test waited in queueu to be picked up" .. a little too many typos :)